examples/sample_run_local_tools_groq.py

```diff
@@ -0,0 +1,60 @@
+"""Groq Sample"""
+import os
+from typing import List
+from dotenv import load_dotenv
+from groq import Groq
+from toolhouse import Toolhouse
+load_dotenv()
+
+TOKEN = os.getenv("GROQCLOUD_API_KEY")
+TH_TOKEN = os.getenv("TOOLHOUSE_BEARER_TOKEN")
+
+
+local_tools = [
+    {'type': 'function',
+     'function':
+         {
+             'name': 'hello',
+             'description': 'The user receive a customized hello message from a city and return it to the user',
+             'parameters': {
+                 'type': 'object',
+                 'properties': {
+                     'city': {'type': 'string', 'description': 'The city where you are from'}
+                 }},
+             'required': ['city']
+         }}]
+
+th = Toolhouse(access_token=TH_TOKEN, provider="openai")
+th.set_metadata("id", "fabio")
+th.set_metadata("timezone", 5)
+
+
+@th.register_local_tool("hello")
+def whatever(city: str):
+    """Return Local Time"""
```

Comment:
Change this so it’s not confusing to the user and it describes the function for what it actually does.